src/components/Auth/NewPasswordModal.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC, memo } from 'react';
+import { faLock } from '@fortawesome/free-solid-svg-icons';
+
+import Modal from '../elements/Modal';
+import TextInput from '../elements/TextInput';
```

Comment:
Use absolute imports here (and in all of the modals)
```suggestion
import Modal from 'components/elements/Modal';
import TextInput from 'components/elements/TextInput';
```

Reply:
Oops, fixed all these imports.